emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
```

Comment:
Do we assume that every common trip will be between two common places? Not an unreasonable assumption, but think through whether that is always true.